ui/button.reel/button.js

```diff
@@ -211,12 +212,47 @@ var Button = exports.Button = Control.specialize(/** @lends module:"montage/ui/n
         }
     },
 
+    __spaceKeyComposer: {
+        value: null
+    },
+
+    _spaceKeyComposer: {
+        get: function () {
+            if (!this.__spaceKeyComposer) {
+                this.__spaceKeyComposer = KeyComposer.createKey(this, "space", "space");
+            }
+            return this.__spaceKeyComposer;
+        }
+    },
+
+    _enterKeyComposer: {
+        get: function () {
+            if (!this.__enterKeyComposer) {
+                this.__enterKeyComposer = KeyComposer.createKey(this, "enter", "enter");
+            }
+            return this.__enterKeyComposer;
+        }
+    },
+
     // HTMLInputElement/HTMLButtonElement methods
     // click() deliberately omitted (it isn't available on <button> anyways)
 
     prepareForActivationEvents: {
         value: function () {
             this._pressComposer.addEventListener("pressStart", this, false);
+            this._spaceKeyComposer.addEventListener("keyPress", this, false);
+            this._enterKeyComposer.addEventListener("keyPress", this, false);
+        }
+    },
+
+    handleKeyPress: {
+        value: function (mutableEvent) {
+            // when focused action event on spacebar & enter
+            // FIXME - property identifier is not set on the mutable event
+            if (mutableEvent._event.identifier === "space" || "enter") {
```

Comment:
here the condition is always equal to true. It should be `mutableEvent._event.identifier === "space" ||  mutableEvent._event.identifier === "enter"`